src/assets/i18n/fr.json5

```diff
@@ -3476,7 +3476,7 @@
 
   // "explore.index.dc.date.accessioned" : "Recent Additions",
   // TODO New key - Add a translation
-  "explore.index.dc.date.accessioned": "Recent Additions",
+  "explore.index.dc.date.accessioned": "Dépots récents",
```

Comment:
TODO :: We could create a PR in "DSpace" original repository to fix some french translation